coops-ui/tests/unit/FilingHistoryList.spec.ts

```diff
@@ -20,7 +20,8 @@ const sampleFilings = [
         'paymentToken': 123,
         'certifiedBy': 'Full Name 1',
         'filingId': 321,
-        'availableOnPaperOnly': false
+        'availableOnPaperOnly': false,
+        'effectiveDate': '2019-06-02T19:22:59.003777+00:00'
```

Comment:
I think these are not legitimate Effective Dates that the FE would actually get from the BE. Wouldn't the time portion actually be T08:01:00 ? (Or T07:01:00 if they filed during Daylight time.)

Reply:
Effective dates for those filing types in these test cases would apply immediately. I just added that property to the test cases as it wasn't a property that existed possibly at the time of the tests creation.

Reply:
Oh, right.
Then it's the time in the test for the subject filing type that should not be T00:00.